packages/app-mobile/components/ButtonWithTooltip.tsx

```diff
@@ -0,0 +1,197 @@
+//
+// A button with a long-press action. Long-pressing the button displays a tooltip
+//
+
+const React = require('react');
+import { ReactNode } from 'react';
+import { themeStyle } from '@joplin/lib/theme';
+import { Theme } from '@joplin/lib/themes/type';
+import { useState, useMemo, useCallback, useRef } from 'react';
+import {
+	View, Text, Pressable, ViewStyle, PressableStateCallbackType,
+	StyleProp, StyleSheet, LayoutChangeEvent, LayoutRectangle, Animated, AccessibilityState, AccessibilityRole,
+} from 'react-native';
+import { Menu, MenuOptions, MenuTrigger, renderers } from 'react-native-popup-menu';
+
+type ButtonClickListener = ()=> void;
+interface ButtonProps {
+	onClick: ButtonClickListener;
+
+	// Accessibility label and text shown in a tooltip
+	description: string;
```

Comment:
If it's a generic button, this should be optional.

Reply:
Good catch! I'll fix that!